src/lib.rs

```diff
@@ -1901,7 +1902,7 @@ pub trait Itertools : Iterator {
     /// assert_eq!(lookup[&3], vec![13, 33]);
     /// ```
     #[cfg(feature = "use_std")]
-    fn into_group_map<K, V>(self) -> HashMap<K, Vec<V>>
+    fn into_group_map<K, V>(self) -> FnvHashMap<K, Vec<V>>
```

Comment:
Now it won't matter in this PR, but we'd usually be mindful here - the function signature changes, so it's a breaking change.

Reply:
Thanks for the review! I completely understand the aversion to adding more dependencies. I would like to note that `FnvHashMap` is just a type alias, so the change would technically be from `HashMap<K, Vec<V>>` to `HashMap<K, Vec<V>, FnvBuildHasher>`, so still a signature change but the hasher is more of an implementation detail.